### DISCUSSION TITLE
Problem with M3 & M4 key binding function

### DISCUSSION THREAD

**Discussion by JohnsonChenz** [seerge/g-helper#209] [Q&A] [answered]
May i ask how can i binding M3 or M4 key with insert key function?

It looks like it can only be used to execute exe or something.

Is there anything i didn't figure out?

**Comment by seerge** [comment]
I didn't get what you mean. Do you want M3 to trigger Insert keypress?

**Reply by JohnsonChenz** [reply]
Yep. i want M3 to trigger Insert keypress,but i don't know how to setup that in key binding settings.

sry for my bad english.

**Reply by seerge** [reply, marked as answer]
You need to set value to 2d (or 0x2d) which is code for insert from https://learn.microsoft.com/en-us/windows/win32/inputdev/virtual-key-codes

![Screenshot 2023-04-09 130914](https://user-images.githubusercontent.com/5920850/230769075-ff49ee8b-fe66-4b35-8cb3-eeff22f1749b.png)

**Reply by JohnsonChenz** [reply]
Got it, works like a charm!

Thank you very much for your help!
